src/Utils/Helpers.spec.ts

```diff
@@ -38,4 +43,146 @@ describe('Helpers', () => {
       ]);
     });
   });
+
+  describe('wildcardToRegExp', () => {
+    [
+   //   { wildcard: '*.example.com', input: 'example.com', expected: true },
+      { wildcard: '*.example.com', input: 'sub.example.com', expected: true },
+      { wildcard: '*.example.com',input: 'sub.sub.example.com', expected: true },
+      { wildcard: '*.example.com',input: 'examp1e.com', expected: false },
+      { wildcard: '*.example.com',input: 'example.co', expected: false },
+      { wildcard: '*',input: 'sub.example.co', expected: true },
+      { wildcard: '*',input: 'sub.sub.example.co', expected: true },
+      { wildcard: '*',input: 'example.com', expected: true },
+
+    ].forEach(({wildcard,  input, expected }) => {
+      it(`regexp '${wildcard}' tests '${input}' ${expected ? 'positive' : 'negative'}`, () => {
+        // arrange
+        const regex = Helpers.wildcardToRegExp(wildcard);
+        // act
+        const test = regex.test(input);
+        // assert
+        test.should.be.equal(expected);
+      });
+    });
+  });
+
+  describe('selectEnumValue', () => {
+    it('should found with case agnostic', () => {
+      //arrange
+      //act
+      const actual = Helpers.selectEnumValue(TestEnum, 'TesT');
+      //assert
+      actual.should.be.equal(TestEnum.TEST);
+    });
+    it('not found should return undefined', () => {
```

Comment:
Please follow test suites/cases naming convention.

- **Unit tests**:

```ts
describe('Class name', () => {
  describe('method name', () => {
    it('should do someting', () => {
       // code
    });
  })
})
```

_Example_: see above

- **e2e/integration**:

```ts
describe('Operation name', () => {
  describe('somehow', () => {
    it('should do someting', () => {
       // code
    });
  })
})
```

_Example_:

```ts
describe('Run repeater', () => {
  describe('with --daemon option', () => {
    it('should run repeater in deamon mode', () => {
       // code
    });
  })
})
```

```suggestion
    it('should return undefined', () => {
```